VaticanBAV/et/BAVet151.xml

```diff
@@ -89,16 +91,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
 <list>
 
+<item xml:id="a1">
```

Comment:
If you know that these are later additions, everything is fine, but if they might also have been included during the original production of the manuscript, I would change the addition type to e.